src/rpc/mining.cpp

```diff
@@ -500,12 +500,13 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
                             {"mode", RPCArg::Type::STR, /* treat as named arg */ RPCArg::Optional::OMITTED_NAMED_ARG, "This must be set to \"template\", \"proposal\" (see BIP 23), or omitted"},
                             {"capabilities", RPCArg::Type::ARR, /* treat as named arg */ RPCArg::Optional::OMITTED_NAMED_ARG, "A list of strings",
                                 {
-                                    {"support", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "client side supported feature, 'longpoll', 'coinbasetxn', 'coinbasevalue', 'proposal', 'serverlist', 'workid'"},
+                                    {"str", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "client side supported feature, 'longpoll', 'coinbasevalue', 'proposal', 'serverlist', 'workid'"},
```

Comment:
nit: I would have named this `feature` to be more expressive

Reply:
fixed

Reply:
The rest of our RPC doc uses "str" for things like this...

Reply:
Where can I find that? Maybe I am doing something wrong but `git grep "RPCArg::Type::STR" | grep "\"str\""` gives me no results.